src/index.js

```diff
@@ -99,6 +99,28 @@ export default function parse(md, prevLinks) {
 		else if (token[17] || token[1]) {
 			chunk = tag(token[17] || '--');
 		}
+		// Table parser
+		else if (token[18]) {
+			var l = token[18].split('\n'),
+					i = l.length,
+					table = '',
+					r = 'td>';
+			while ( i-- ) {
```

Comment:
might want to remove the spaces inside the parentheses for consistency with the rest of the file

Reply:
and to save some bytes ofc ;)

Reply:
removed them for code-styles sake
@tbjgolden since it's minified spaces don't add to final size